### ISSUE TITLE
After waking system from sleep mode, soundflower changes from user selection from Builtin-Output to None(off)

### ISSUE THREAD

**Issue by GoogleCodeExporter** [mattscotthudson/soundflower#35]
```
What steps will reproduce the problem?
1. Set output of soundflower to built-in output (had system output to 
soundflower 2 channel)
2. Invoke system to sleep
3. wake system

What is the expected output? That the selected before machine went to sleep 
would still be chosen 
after waking.

What do you see instead? That soundflower was sending the output to None(Off), 
so no sound 
comes from machine.

What version of the product are you using? 1.5.0 dmg (in menu reports as 1.4.4)

On what operating system? Snow Leopard



Please provide any additional information below.

This had been an issue in the previous version on Leopard.

Much thanks for the quick update after snow leopard released.

```

Original issue reported on code.google.com by `wells.lu...@gmail.com` on 2 Sep 2009 at 5:07
- Merged into: #18

**Comment by GoogleCodeExporter** [comment]
```
I am a little new to OSX programing API so this may be completely off, but I 
think - (void)awakeFromNib may be 
getting called on the system waking thus setting the menu back to the default 
after the application loads, could 
the code in their be moved instead to + (void)initialize

It would defiantly still loose the settings if the machine were turned off, but 
might retain the menu state without 
having much to change to store/relive settings while the machine sleep and woke.
```

Original comment by `wells.lu...@gmail.com` on 2 Sep 2009 at 11:59

**Comment by GoogleCodeExporter** [comment]
Original comment by `74obje...@gmail.com` on 29 Sep 2009 at 12:47
- Changed state: **Accepted**
- Added labels: **Priority-High**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
I have the same Problem. This sux! Is there a solution now?
```

Original comment by `julian.s...@googlemail.com` on 31 Oct 2009 at 7:22

**Comment by GoogleCodeExporter** [comment]
```
I got the same problem! Please help! 
```

Original comment by `derdigge...@gmail.com` on 5 Nov 2009 at 11:11

**Comment by GoogleCodeExporter** [comment]
```
I am seeing the exact same problem on a 15" MBP late 2009 running 10.6.2. Quite 
annoying. Works fine on my 
Mac Pro 2008 running 10.6.2 though...
```

Original comment by `v1so...@gmail.com` on 13 Nov 2009 at 11:48

**Comment by GoogleCodeExporter** [comment]
```
Same issue here.  Would really like it to remember my setting not only after 
sleep but also after shutdown.  Why 
can't I set the output and leave it?
```

Original comment by `troydona...@gmail.com` on 24 Nov 2009 at 4:15

**Comment by GoogleCodeExporter** [comment]
```
*agree* same problem here! Please try to fix it, thank you,
```

Original comment by `TheDomi...@gmail.com` on 24 Nov 2009 at 1:38

**Comment by GoogleCodeExporter** [comment]
```
Same problem here. PLEASE FIX THIS ASAP!  I'd be willing to donate to the cause 
if necessary.  It should be a 
simple fix.
```

Original comment by `garb...@gmail.com` on 27 Nov 2009 at 2:37

**Comment by GoogleCodeExporter** [comment]
```
I have this problem too, please fix it!  It's annoying having to reset up the 
audio every time I wake up my MacPro!
```

Original comment by `slice...@gmail.com` on 27 Nov 2009 at 5:50

**Comment by GoogleCodeExporter** [comment]
```
Any word on this yet?
```

Original comment by `troydona...@gmail.com` on 11 Dec 2009 at 2:27

**Comment by GoogleCodeExporter** [comment]
```
Yup, any word on this sleep issue?  Annoying issue.
Power mac G4 Tunnel, Leopard 10.5.8
```

Original comment by `iam...@gmail.com` on 13 Dec 2009 at 10:07

**Comment by GoogleCodeExporter** [comment]
```
Same here too; 'None(Off)' instead of 'built-in output' after wake up. 

In addition; soundflower prevents my Mac going to sleep after the idle time 
(set in
PreferencePane EnergySaver).


MacMini C2D, MacOS 10.4.11, Soundflower 1.4.3
```

Original comment by `detlef.m...@googlemail.com` on 17 Feb 2010 at 10:41

**Comment by GoogleCodeExporter** [comment]
Original comment by `74obje...@gmail.com` on 8 Mar 2010 at 5:27
- Changed state: **Duplicate**

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
The fix with the new AppController.mm from id=18 did not work for me, after 
sleep it
again goes back to None(OFF).  I just rebuilt SoundflowerBed.app with xcode 
3.1.2 on
a Power mac G4 Tunnel, Leopard 10.5.8 and double-checked.  Anyone else with 
better
results ?
```

Original comment by `iam...@gmail.com` on 11 Mar 2010 at 12:42

**Comment by GoogleCodeExporter** [comment]
```
This should be unmerged from issue 18 and re-opened as it is a different issue. 
 The
fix for 18 does not work for issue 35 - have anyone else tried a rebuild of the 
new
AppController.mm with an Intel device - it sure does not work for the powermacs.
```

Original comment by `iam...@gmail.com` on 23 Mar 2010 at 6:06

**Comment by GoogleCodeExporter** [comment]
```
I too am experiencing the same issue - after wake no audio output - IMAC intel, 
OS X SL.
```

Original comment by `donald.g...@gmail.com` on 11 Sep 2010 at 7:34

**Comment by GoogleCodeExporter** [comment]
```
I am still having this issue running v. 1.5.2 (after sleep or reboot) on 
SnowLeopard 10.6.7, but it doesn't seem 100% consistent.  

If you need any logs or anything let me know.


```

Original comment by `bleeker....@gmail.com` on 17 Jun 2011 at 5:11

**Comment by GoogleCodeExporter** [comment]
```
This exact issue (nota bene: this is the one where Soundflower is still running 
and »None (Off)« is selected in the menu after waking from sleep) still 
occurs with 1.6.6 on 10.8. I would appreciate it if this could be addressed.
```

Original comment by `jannis.s...@gmail.com` on 13 Jan 2013 at 11:04
